subhosting/manual/pricing_and_limits.md

```diff
@@ -0,0 +1,23 @@
+# Pricing and Limits
+
+## Deployment size
+
+Deployments should be less than 1GB across all source code and assets in aggregate, per deployment.
+
+## Deployment frequency
+
+Bert to define, varies based on pricing tier. 
```

Comment:
The maximum number of deployments per hour that a subhoster can make is either 60 (free tier) or 300 ("builder" tier). Higher limits are available for organizations on the enterprise plan.